app/model.py

```diff
@@ -629,6 +629,13 @@ def generate_by_person_record_id(
             query.with_cursor(query.cursor())  # Continue where fetch left off.
             notes = query.fetch(Note.FETCH_LIMIT)
 
+    @staticmethod
+    def get_unreviewed_record_count(repo, filter_expired=True):
+        """Gets the number of unreviewed notes."""
+        query = Note.all_in_repo(repo, filter_expired=filter_expired
+            ).filter('reviewed =', False)
```

Comment:
Add hidden = False condition here as well. It will filter out notes already marked as spam.